ExampleApp/ExampleApp/ViewController.swift

```diff
@@ -79,4 +82,12 @@ class ViewController: UIViewController {
         }
     }
     
+    @IBAction func showServiceInstanceClick(_ sender: Any) {
+        serviceX.Passport().showDetail(
+            from: self,
+            user: user,
+            marketId: "8af0e885-a06c-4508-8d17-03e4fa1ea526",
```

Comment:
Please move the hard-coded value to the beginning of the file along with `API_KEY`, etc..

Reply:
I fixed it